backend/plugins/cars_api/src/modules/cars/db/models/carModel.ts

```
@@ -0,0 +1,151 @@
+import { ICar, ICarDocument } from '~/modules/cars/@types/car';
+import { validSearchText } from 'erxes-api-shared/utils';
+import { any } from 'zod';
+
+export interface ICarModel extends Model<ICarDocument> {
+  fillSearchText(doc: ICar);
+  carDetail(_id: string): Promise<ICarDocument>;
+  cars(): Promise<ICarDocument[]>;
+  carsAdd(doc: ICar): Promise<ICarDocument>;
+  carsEdit(_id: string, doc: ICar): Promise<ICarDocument>;
+  carsRemove(ModuleId: string[]): Promise<{ ok: number }>;
+  carDetail(_id: string): Promise<ICarDocument>;
```

ESLint

[error] All carDetail signatures should be adjacent.

(@typescript-eslint/adjacent-overload-signatures)